pkg/minikube/constants/constants.go

```diff
@@ -31,9 +31,9 @@ const (
 	MiniShiftHomeEnv             = "MINISHIFT_HOME" // Environment variable used to change the Minishift home directory
 	VersionPrefix                = "v"
 	MinOpenshiftSupportedVersion = "v1.4.1"
-	DefaultMemory                = 2048
+	DefaultMemory                = "2G"
```

Comment:
So why here '2G' and for disk size '20GB'. What does 'G' map to GB or GiB. We do not do the users a favour by using 'G' or 'M'. There is enough confusion around MB vs MiB and GB and GiB.